src/refmt_args.ml

```diff
@@ -42,7 +42,7 @@ let print =
 let print_width =
   let docv = "COLS" in
   let doc = "wrapping width for printing the AST" in
-  Arg.(value & opt (int) (100) & info ["w"; "print-width"] ~docv ~doc)
+  Arg.(value & opt (int) (90) & info ["w"; "print-width"] ~docv ~doc)
```

Comment:
`reason_pprint_ast.ml` contains `defaultSettings` 90.
Not sure why I automatically changed it here to 90 lol.
I can change it back, your call.